TestViewer/MainPage.xaml.cs

```diff
@@ -0,0 +1,67 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Numerics;
+using System.Threading.Tasks;
+using Windows.Storage;
+using Windows.Storage.Streams;
+using Windows.UI.Composition;
+using Windows.UI.Composition.Scenes;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Hosting;
+using SceneLoaderComponent;
+
+namespace TestViewer
+{
+    /// <summary>
+    /// A page that generates a gltf image using the SceneLoaderComponent.
+    /// </summary>
+    public sealed partial class MainPage : Page
+    {
+        Compositor _compositor;
+        SceneVisual _sceneVisual;
+
+        public MainPage()
+        {
+            this.InitializeComponent();
+
+            _compositor = ElementCompositionPreview.GetElementVisual(this).Compositor;
+
+            var root = _compositor.CreateContainerVisual();
+
+            root.Size = new Vector2(1000, 1000);
+            ElementCompositionPreview.SetElementChildVisual(this, root);
+
+            _sceneVisual = SceneVisual.Create(_compositor);
+            root.Children.InsertAtTop(_sceneVisual);
+
+            _sceneVisual.Offset = new Vector3(300, 300, 0);
+            _sceneVisual.RotationAxis = new Vector3(0, 1, 0);
+
+            var rotationAnimation = _compositor.CreateScalarKeyFrameAnimation();
+
+            rotationAnimation.InsertKeyFrame(0, 0, _compositor.CreateLinearEasingFunction());
+            rotationAnimation.InsertKeyFrame(0.5f, 360, _compositor.CreateLinearEasingFunction());
+            rotationAnimation.InsertKeyFrame(1, 0, _compositor.CreateLinearEasingFunction());
+
+            rotationAnimation.Duration = TimeSpan.FromSeconds(8);
+            rotationAnimation.IterationBehavior = AnimationIterationBehavior.Forever;
+
+            _sceneVisual.StartAnimation("RotationAngleInDegrees", rotationAnimation);
+        }
+
+        async Task<SceneNode> LoadGLTF(Uri uri)
+        {
+            var storageFile = await StorageFile.GetFileFromApplicationUriAsync(uri);
+            var buffer = await FileIO.ReadBufferAsync(storageFile);
+
+            var loader = new SceneLoader();
+            return loader.Load(buffer, _compositor);
+        }
+
+        private async void Page_Loaded(object sender, RoutedEventArgs e) => _sceneVisual.Root = await LoadGLTF(new Uri("ms-appx:///Assets/DamagedHelmet.gltf"));
```

Comment:
>private [](start = 8, length = 7)

Be consistent about your use of accessibility modifiers. LoadGLTF doesn't have a modifer but this one does. You need to decide on what style to use and stick to it. Either always specify accessibility or only specify non-default accessibility.

Reply:
You have to make your fields "private" too if you're choosing to specify accessibility on everything.

---
In reply to: [280939382](https://github.com/windows-toolkit/SceneLoader/pull/1#discussion_r280939382) [](ancestors = 280939382)